src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Ball {
+    private final String color;
+
+    private final int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "Ball{" + "color='" + color
+                + '\'' + ", number=" + number + '}';
+    }

```

Comment:
The class `Ball` should have access modifiers for its fields and methods. By default, they have package-private access, which might not be suitable for all cases. Consider defining explicit access modifiers to adhere to best practices.